kod/object/active/holder/nomoveon.kod

```diff
@@ -513,5 +513,23 @@ messages:
       return iResult;
    }
 
+   TryDeflect(what = $,caster=$)
+   {
+      local oDeflect, lRadiusEnchantment;
+
+      foreach lRadiusEnchantment in plRadiusEnchantments
+      {
+         if Send(Nth(lRadiusEnchantment,1),@TryDeflect,
```

Comment:
I'm very interested in whether you have a better implementation for this entire mechanic. Currently, I often call every Radius Enchantment a player has for most hooks. First, because players very rarely have REs at all, and if they do, only one or a few; second, because it enables me to put all the 'content' into the Radius spell files themselves. I want future contributors to be able to simply create a new spell file, fill in TryDeflect (for example), and have it work without needing to hunt down the hook and make a specific call for their new class.

But I'd definitely like a better way to do this. Only thing I (with my limited skills) can think of is adding a DoesThis boolean to every single type of function. Like "if Send(radius_enchant,@DoesDeflect) { Send(radius_enchant,@TryDeflect }" to try to save a bit of processing time.